toolchain/docs/lower.md

```diff
@@ -23,3 +25,64 @@ can first generate type information for function arguments.
 Lowering is done per `SemIR::InstBlock`. This minimizes changes to the
 `IRBuilder` insertion point, something that is both expensive and potentially
 fragile.
+
+## Mangling
+
+Part of lowering is choosing deterministically unique identifiers for each
+lowered entity to use in platform object files. Any feature of an entity (eg:
+the namespace it appears in, parameters for overloaded functions, etc) that
+would create a distinct entity must be included in some way in the generated
+identifier.
+
+The current rudimentary name mangling scheme is as follows:
+
+-   As a special case, `Main.Run` is emitted as `main`.
+
+Otherwise the resulting name consists of:
+
+1.  `_C`.
```

Comment:
```suggestion
1.  `_C`
```

I feel weird about this. While a period has a degree of consistency, lines 46 and 50 (for example) only put the character and don't have a period. I lean towards it reading better without it.